ProcessMaker/processmaker
# Downgrade psr/http-message Version to 1.1

This PR addresses compatibility issues with the Plaid package dependency `tomorrow-ideas/plaid-sdk-php`. The downgrade is necessary to maintain compatibility with dependencies that require `psr/http-message ^1.0`.

## How to Test

1. Install the Plaid Package
2. Verify that there are no compatibility issues or errors related to `psr/http-message`.

## Related Tickets & Packages
- https://processmaker.atlassian.net/browse/FOUR-13367

ci:next

## Code Review Checklist
